docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,728 @@
+# Audit log filter functions, options and variables
+
+The following sections describe the [functions](#audit-log-filter-functions), [options, and variables](#audit-log-filter-options-and-variables) available in the audit log filter component.
+
+## Audit log filter functions
+
+The following audit log filter functions are available.
+
+| Function name |
+| --- | 
+| [audit_log_encryption_password_get(keyring_id)](#audit_log_encryption_password_getkeyring_id) |
+| [audit_log_encryption_password_set(new_password)](#audit_log_encryption_password_setnew_password) |
+| [audit_log_filter_flush()](#audit_log_filter_flush) |
+| [audit_log_read()](#audit_log_read) |
+| [audit_log_read_bookmark()](#audit_log_read_bookmark) |
+| [audit_log_session_filter_id()](#audit_log_session_filter_id() |
```

Comment:
Not sure but looks like link format is broken here

Reply:
fixed